test/e2e/setup.go

```diff
@@ -377,9 +391,35 @@ var _ = BeforeSuite(func() {
 
 	if err := setup(context.Background()); err != nil {
 		panic(err)
+	} else {
+		m := statsd.New(
+			context.Background(),
```

Comment:
don't we want to have the context passed to the setup function here?

Reply:
Moved logrus hook setup into `setup` function so that the same context could be used.